src/platform/datapath_raw_xdp_linux_kern.c

```diff
@@ -0,0 +1,221 @@
+/* SPDX-License-Identifier: GPL-2.0 */
```

Comment:
We need to be careful if we include GPL license code since MsQuic is MIT license.